fix(shared-data): Change defaultReturnTipHeight values for 96 channel pipettes

caila-marashaj opened the pull request:
## Overview
This closes https://opentrons.atlassian.net/browse/RABR-730.

In shared-data, there's a `defaultReturnTipHeight` value for every pipette/tip combination that represents what fraction of the tip should be submerged into the tiprack before dropping tips. Previously, this was somewhere between `0.6-0.8` for everything except 96 channel pipettes, which were `0.2`. 

It's possible that this incongruity was an effort to overcome some wonky additional z offset calculation, and this behavior started surfacing recently because of (good) labware offset changes. In any case, we need tips to be somewhere between 60-80% inside the tiprack before dropping them, so let's update that.

I chose `0.71` for t1000's because the height for other pipettes' t1000 entries, `0.82`, was too large and caused the pipette to hit the tiprack adapter pegs, so I went with what seems like the standard for the smaller tips in the single and 8 channels' `supportedTips`.
`0.6` is known to work on the 96 channel for t20 tips, so I applied to t50 and t200 as well. All of these are tested and look ok on the robot, but let me know if there are other values we should use instead.

## Review Requests
Which version files should we change?

Closes EXEC-1108
Closes RABR-730

caila-marashaj committed:
change defaultReturnTipHeight for 96 channel

sfoster1 reviewed (approved):
Oh, didn't see the new PR. Looks good once tests pass.

Change all the version files IMO.

caila-marashaj committed:
change all version files for 96ch

caila-marashaj committed:
fix p200 values + step generation test

github-actions[bot] committed:
fix(analyses-snapshot-testing): heal RABR-730-drop-tip-location-fix snapshots (#17514)

This PR was requested on the PR
https://github.com/Opentrons/opentrons/pull/17498

Co-authored-by: caila-marashaj <98041399+caila-marashaj@users.noreply.github.com>

caila-marashaj committed:
revert snapshot changes

github-actions[bot] committed:
fix(analyses-snapshot-testing): heal RABR-730-drop-tip-location-fix snapshots (#17515)

This PR was requested on the PR
https://github.com/Opentrons/opentrons/pull/17498

Co-authored-by: caila-marashaj <98041399+caila-marashaj@users.noreply.github.com>
